### ISSUE TITLE
Error in admin with UserAdmin

### ISSUE THREAD

**Issue by neokeats** [s-block/django-nested-inline#51]
Hi, 

i had to comment line 183 of nested_inline/admin.py to get it to work 

``` python
 if self.all_valid_with_nesting(formsets) and form_validated:
                self.save_model(request, new_object, form, False)
                self.save_related(request, form, formsets, False)
                #self.log_addition(request, new_object)
                return self.response_add(request, new_object)
```

``` python
class UserAdmin(BaseUserAdmin, NestedModelAdmin):
    list_display = ('username', '__str__', 'is_staff', 'is_active' )
    inlines = [VendorInline]

admin.site.unregister(User)
admin.site.register(User, UserAdmin)
```

**Comment by ppython** [comment]
Hey, just saw this is linked to issue #52 and PR #53 seems to fix your problem :)

**Comment by daniel-brenot** [comment]
Closing since this has been confirmed fixed in PR #53
